hazelcast/src/main/java/com/hazelcast/config/ReplicatedMapConfig.java

```diff
@@ -54,10 +53,6 @@
      * Default value of asynchronous fill up
      */
     public static final boolean DEFAULT_ASNYC_FILLUP = true;
-    /**
-     * Default policy for merging
-     */
-    public static final String DEFAULT_MERGE_POLICY = PutIfAbsentMapMergePolicy.class.getName();
```

Comment:
I think we cannot remove this, since it's a public constant on a public config class :(

But we should change its value.

Reply:
fixed